app/styles/themes/white-label/_theme.overrides.scss

```diff
@@ -281,12 +296,14 @@
 
 .c-newsletter-form__terms input[type="checkbox"] {
   background-color: rgb(var(--color-ceramic-blue));
-  border: 2px solid rgb(var(--color-white));
+  border: 2px solid;
+  border-color: rgb(var(--color-white));
 }
 
 .c-newsletter-form__terms input[type="checkbox"]:checked {
   background-color: rgb(var(--color-ceramic-blue));
-  border: 2px solid rgb(var(--color-white));
+  border: 2px solid;
+  border-color: rgb(var(--color-white));
```

Comment:
I think I had to make this same fix a while back for the same reason -- worked locally, but not on demo. It looked to me like the color was getting dropped when the CSS was compiled and compressed, not sure why.